src/tag_version/core.py

```
@@ -0,0 +1,160 @@
+        message: Optional message for annotated tag. If None, creates lightweight tag.
+
+    Returns:
+        True if tag creation was successful
+
+    Raises:
+        RuntimeError: If tag creation fails
+    """
+    try:
+        if message:
+            # Create annotated tag with message
+            subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)